docs/_posts/2019-03-23-header-config-cn.md

```diff
@@ -0,0 +1,139 @@
+---
+category: demo
+tags:
+  - cn
+date: 2019-03-23
```

Comment:
The `date`s in the docs are not for the exact created time, but for the display order.

(we could also support `top: true` frontmatter to make a certain post placed on the top)

Reply:
ok, i'll use `2019-01-20`, the day before the en, to keep the same part together.